lesson27/src/js/index.js

```diff
@@ -3,6 +3,8 @@ import * as loading from "./module/loading";
 import * as slide from "./contents/slideshow";
 import * as tab from "./contents/tab";
 
+if(!localStorage.getItem("token")) window.location.href = "./login.html";
+
```

Comment:
On the contents page, check `token`.
If the TOKEN does not exist, the user will be redirected to the LOGIN page.